scenarios/tracking/02_mot_challenge.ipynb

```diff
@@ -0,0 +1,463 @@
+{
```

Comment:
<p>Not sure what this comment here (and in cell below) means:</p><pre class="ql-syntax" spellcheck="false"># run_eval: if need to re-run the prediction
</pre><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/computervision-recipes/pull/569/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
Deleted run_eval